collect_app/src/main/java/org/odk/collect/android/activities/SplashScreenActivity.java

```diff
@@ -66,9 +60,6 @@ private void init() {
 
         if (showSplash) {
             startSplashScreen(splashPath);
-
-            String splashPathHash = FileUtils.getMd5Hash(new ByteArrayInputStream(splashPath.getBytes()));
-            analytics.logEvent(SHOW_SPLASH_SCREEN, splashPathHash, "");
```

Comment:
Let's ask @lognaturel I think we kept that even long enough to collect data we and make a decision to keep or remove that screen.

Reply:
Yes, let's get rid of the analytics for now. The splash screen is not a hugely popular feature (~5% of users see a splash screen) but a few big projects seem to make use of it. I think it's worth maintaining for now and we can bring the analytics back again if we see a strong reason to remove. @grzesiek2010 made the point that it might be kind of nice in a multi-project context because then each project can have its own splash screen which gives users an expectation of what context they're in before Collect starts.

Reply:
Sounds good! We should just delete the constant in `AnalyticsEvents` to finish the clean up then.

Reply:
Ok I removed that event.